web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx

```diff
@@ -278,6 +284,41 @@ const ViewFeedbackResponses = () => {
         ))}
       {!isLoading &&
         filteredQuestionsAndAnswers?.map(question => {
+          // Handle Section Headers for questions 3 and 10
+          if (question.questionNumber === 3) {
+            return (
+              <Box
+                key={`section-1`}
+                border={1}
+                borderColor="gray"
+                p={2}
+                mb={3}
+                style={{ fontSize: '1.2rem', fontWeight: 'bold' }}
+              >
+                Section 1: How often has this team member displayed each of the following behaviors during the period covered by this review?
+              </Box>
+            );
+          }
+
+          if (question.questionNumber === 10) {
+            return (
+              <Box
+                key={`section-2`}
+                border={1}
+                borderColor="gray"
+                p={2}
+                mb={3}
+                style={{ fontSize: '1.2rem', fontWeight: 'bold' }}
+              >
+                Section 2: You can provide more detailed context to your review of this team member below. Be as specific as possible!
+              </Box>
+            );
+          }
+
+          // Handle other questions by removing "Q#: "
+          const questionText = question.question.replace(/^Q\d+: /, '');
+          const hasResponses = question.answers.length > 0;
+
```

Comment:
The above changes are problematic. They assume that the template used for reviews is static, but that is not the case.  Doing this creates an even tighter coupling to the data (template questions) than we already have. I would roll this bit back and stick to just addressing the items mentioned in the issue.

Reply:
Done.